src/sentry/event_manager.py

```diff
@@ -687,7 +688,7 @@ def stringify(f):
         elif '_meta' in data:
             del data['_meta']
 
-        self._data = prune_empty_keys(data)
+        self._data = CanonicalKeyDict(prune_empty_keys(data))
```

Comment:
I don't think this does anything but we traditionally return `CanonicalKeyDict` from `EventManager`